config/docs/options_README.md

```diff
@@ -0,0 +1,264 @@
+# Documentation: Options Configuration
+
+Detailed information on how to customize the `options.yaml` for each Pooled Cell Painting experiment.  
+
+For more information on `.yaml`, read the [YAML page on Wikipedia](https://en.wikipedia.org/wiki/YAML).
+
+Structure is very important in `.yaml` files.
+When editing the `.yaml` make sure to maintain dashes and indentations.
+(When information is added on the same line, it makes a dictionary where the value is a string.
+When there are multiple lines, the value is a list with string elements.)
+
+## core:  
+
+`compartments:` *The cellular compartments identified as objects in the cell painting experiment.  
+Default cell painting compartments are:  
+e.g.*  
+    - Cells  
+    - Nuclei  
+    - Cytoplasm  
+
+`cell_quality:`  
+`categorize_cell_quality:` *The cell categorization method you would like to use. The methods are described in cell_quality_utils.py in the Pooled Cell Painting Profiling Recipe.  
+e.g.* simple  
+*e.g.* simple_plus  
+`cell_filter:`*Select the categories of cells that you wish to use for analysis. Categories are defined in cell_quality_utils.py in the Pooled Cell Painting Profiling Recipe.
+Make sure that the options you set here are available in the categorize_cell_quality method that you set above in core config.  
+e.g.*  
+    - Perfect  
+    - Great  
+`cell_quality_column:` *The column that contains the numerical cell quality category.  
+e.g.* Metadata_Foci_Cell_Quality  
+`cell_quality_index:` *The column that contains DESCRIBETHIS  
```

Comment:
@gwaygenomics

Reply:
oof, this is a tough one to describe. It contains info on how to rename the index in the cell quality dataframe (see https://github.com/broadinstitute/pooled-cell-painting-profiling-recipe/blob/8f489c156c375f5dee542c578c8a31495697ea20/scripts/cell_quality_utils.py#L62).